src/Stratis.Bitcoin.Features.Miner/MiningFeature.cs

```diff
@@ -241,9 +240,7 @@ public static IFullNodeBuilder AddPowPosMining(this IFullNodeBuilder fullNodeBui
                         services.AddSingleton<IAssemblerFactory, PosAssemblerFactory>();
                         services.AddSingleton<MinerController>();
                         services.AddSingleton<MiningRPCController>();
-                        services.AddSingleton<IWalletManager, WalletManager>();
```

Comment:
I think you need the walletManager for mining, don't you?

Reply:
Well WalletFeature is a dependency so it should already be registered in the DI so this call is actually redundant